grpc-gateway/luceneserver.swagger.json

```diff
@@ -299,6 +299,39 @@
         ]
       }
     },
+    "/v1/get_nodes_info": {
```

Comment:
I don't see this specified in the proto files. Was something missed ? There are some other inconsistencies too.

Reply:
Resolving since this got fixed right after I reviewed